qiskit_dynamics/models/rotating_frame.py

```diff
@@ -610,28 +595,28 @@ def vectorized_map_into_frame(
         return op
 
 
-def _is_herm_or_anti_herm(mat: Array, atol: Optional[float] = 1e-10, rtol: Optional[float] = 1e-10):
+def _enforce_anti_herm(mat: Array, atol: Optional[float] = 1e-10, rtol: Optional[float] = 1e-10):
```

Comment:
Should this be ``_enforce_herm_or_anti_herm``?

Reply:
IMO _enforce_anti_herm is correct because this function enforces that the output be anti-Hermitian with any input. _is_herm_or_anti_herm makes it sound like it'll return True or something if the input is either. _enforce_herm_or_anti_herm sounds like it will enforce one of those two things.